vic/drivers/image/src/vic_mpi_support.c

```diff
@@ -1742,7 +1744,7 @@ gather_put_nc_field_double(char   *nc_name,
 
     if (mpi_rank == 0) {
         grid_size = global_domain.n_nx * global_domain.n_ny;
-        dvar = (double *) malloc(sizeof(double) * grid_size);
+        dvar = malloc(sizeof(*dvar) * grid_size);
```

Comment:
In the previous we almost always use `number_of_items * size`, probably to make it look more like the `calloc()` call. May be worth changing to make consistent, here and in the next few calls.